app/lib/config/config.go

```diff
@@ -33,6 +33,8 @@ type CothoritydConfig struct {
 	Public  string
 	Private string
 	Address network.Address
+	TLSCert string
+	TLSKey  string
```

Comment:
And with regard to the private key - this is the configuration-file of the server, so I think it's OK to store the private key in there. The Conode-private key is just 3 lines above...